tests/ao-dfcasscf-sp/input.dat

```diff
@@ -0,0 +1,21 @@
+#! CASSCF/6-31G** energy point
+
+molecule {
+O
+H 1 1.00
+H 1 1.00 2 103.1
+}
+
+set {
+    basis           6-31G**
+    df_basis_scf    cc-pvdz-jkfit
+    reference       rhf
+    frozen_docc     [1, 0, 0, 0]
+    active          [3, 0, 1, 2] 
+    scf_type        df
+    mcscf_type      ao
+}
+
+casscf_energy = energy('casscf')
+
+compare_values(-76.073736807922970, casscf_energy, 6, 'AO-DFCASSCF Energy')  #TEST
```

Comment:
And I still think it's nice to put in a SCF energy check so that when it breaks you'll have a quick indicator of total catastrophe vs. ao-dfcasscf trouble.